src/components/Icon/Icon.stories.tsx

```diff
@@ -1,7 +1,27 @@
 import React from "react"
 import { storiesOf } from "@storybook/react"
 import Icon from "./index"
+import * as Icons from "./Icons"
+
+const allIcons = Object.values(Icons)
 
 storiesOf("Icon", module)
   .add("Basic Icon", () => <Icon type="oAddCircle" />)
   .add("Icon with color", () => <Icon type="oAddCircle" fill="red" />)
```

Comment:
Can we have one or two stories with knobs for the key attributes like `heght`, `width`, and `color`?

Reply:
Added the knob for size.